picard/i18n.py

```diff
@@ -55,13 +92,16 @@ def _init_default_locale():
         defaults = Foundation.NSUserDefaults.standardUserDefaults()
         current_locale = defaults.objectForKey_('AppleLanguages')[0]
         current_locale = current_locale.replace('-', '_')
-        locale.setlocale(locale.LC_ALL, current_locale)
-        return current_locale
+        try:
+            locale.setlocale(locale.LC_ALL, current_locale)
+            return current_locale
```

Comment:
It would be preferable to return the actual locale set by `locale.setlocale()` but I see there's a `replace()` above, so not sure what it is returning on macos.